content/docs/learning-guide/tutorials/environments/generate-terrain.md

```diff
@@ -0,0 +1,303 @@
+
+You might, however, have issues where the valleys and peaks in your terrain appear flat due to clipping. The following image shows a terrain that has been placed so that the bottom of its box shape falls below the **Min Height** specified in the **Terrain World** level component:
+
+{{< image-width src="/images/learning-guide/tutorials/environments/terrain-clipping-example.png" width="800" alt="An example one square kilometer terrain demonstrating clipping due to entity placement." >}}
+
+The following properties are closely related and might require special attention if you have issues with your terrain such as clipping:
+
+* The **Min Height** and **Max Height** properties of the **Terrain World** level component define the vertical range in the level for terrain generation. This range should be large enough to encompass the vertical extents of the box shape of the Terrain Spawner entity. If the box shape extends below the **Min Height** value or above the **Max Height** value, the terrain valleys or peaks are clipped.
+
+* The **Dimensions** property of the **Axis Aligned Box Shape** component specifies the terrain area. The **Dimensions: Z** property, in particular, specifies the height of the terrain. Its value should not exceed the range defined in the **Terrain World** level component by the **Min Height** and **Max Height** properties.
+
+* The **Translate** property of the **Transform** component of the Terrain Spawner entity places the terrain entity in the level. The translate position of the Terrain Spawner entity is at the center of the box shape that defines the terrain area. This means that if the box shape is a 512 unit cube, and the **Translate** property is `0.0` on each axis, then the terrain extends 256 units in each direction from the origin of the level. Care should be taken to ensure that the **Translate: Z** value places the box shape within the range defined by the **Min Height** and **Max Height** properties of the **Terrain World** level component.
+
+## Apply terrain materials
+
+There are two types of materials that are needed for terrain, the *macro material* and *detail materials*.
+
+The macro material is a basic material that applies to the entire terrain. It supports a simple color texture and a normal texture. The macro material is displayed on terrain that is far from the camera.
+
+Detail materials are standard materials that you can create with [Material Editor](/docs/atom-guide/look-dev/materials/material-editor/). They are assigned to the terrain through surface tag names that are generated by gradients, similar to how this tutorial used an image gradient to create elevation. This allows you to assign many materials across a large terrain surface and blend between them. Detail materials are displayed on terrain surfaces that are close to the camera.
```

Comment:
The macro material provides low-fidelity color and normal information across the entire terrain, both near and far. The detail materials provide high-fidelity color and normal information that is blended together with the macro material for terrain surfaces close to the camera.

The current wording makes it sound like _only_ the detail materials are used for near-field rendering.

To help explain this, one of the problems with terrain creation is that you want infinite variety and uniqueness across the entire terrain with a high level of detail, but creating a single unique high-resolution PBR material across a large terrain would consume far too much storage. So instead, a lower-resolution texture (the "macro material") is used to describe general color variation across the terrain, and a handful of high-resolution materials (the "detail materials") are blended and repeated across the terrain for high-fidelity details. By combining the two, you can have a single grass detail material that's repeated across the terrain and shows individual grass blade details, but the macro material can provide the green to yellow color variations across a field so that the grass doesn't look uniform.

Reply:
Rewrote this section and included the example images.